api/proto/info/info.proto

```diff
@@ -333,21 +433,65 @@ message ZInfoStorage {
   bool storageLocation = 4; // Storage location for app disks, images etc.
 }
 
+// ZInfoAppNetworkInterface
+//  Status of Network interface used by the app.
+message ZInfoAppNetworkInterface {
+    // appInterfaceName - Name of the interface specified in App Bundle.
+    string appInterfaceName = 1;
+
+    // networkInstanceId - Network instance attached to the interface
+    string networkInstanceId = 2;
+    string networkInstanceName = 3;
+
+    // macAddr - MacAddress of the interface
+    string macAddr = 4;
+    // Ip Address(s) of the interface
+    repeated string IPAddrs = 5; // All IP addresses with /N for subnet
+    repeated string defaultRouters = 6; // If DHCP assigned
+    ZInfoDNS dns = 7; // If DHCP assigned
+    bool up = 8;      // operational up/down status.
+
+    // XXX - Should we have Acl / Rule information here? May be even
+    //  IpTable rules we program?
+}
+
+// ZInfoAppAssignedAdapter - App information for each Direct-Attach adapter
+message ZInfoAppAssignedAdapter {
+    // appAdapterName - name of the Adapter as specified in the App Bundle.
+    string appAdapterName = 1;
+    string physicalLabel = 2;
+    string logicalLabel = 3;
+}
+
 message ZInfoApp {
   string AppID = 1;
   string appVersion = 2;
 
+  // XXX - What is systemApp ?? We always seem to set this to False? Is it for
+  //    future where Dom0 itself runs on multiple Containers?
```

Comment:
That was the intent. We might have reported dom0 as an app a long time ago as well and used this.